driver/src/price_estimation/kraken.rs

```diff
@@ -42,8 +43,8 @@ where
         //   complicated is determining when the cache needs to be invalidated
         //   as new assets get added to Kraken.
 
-        let assets = self.api.assets()?;
-        let asset_pairs = self.api.asset_pairs()?;
+        let assets = self.api.assets().wait()?;
+        let asset_pairs = self.api.asset_pairs().wait()?;
```

Comment:
nit: would it be hard to join these to have them run in parallel (I realize this wasn't the behavior before but could be a nice and simple improvement)

Reply:
It would be easy. I thought about this too but didn't want to change the behavior yet. I will change it when making price estimation async.